src/PolykeyAgent.ts

```diff
@@ -316,6 +326,15 @@ class PolykeyAgent {
         keyRing,
         logger: logger.getChild(NodeGraph.name),
       });
+      mdns = new MDNS({
+        logger: logger.getChild(MDNS.name)
+      });
+      mdns.start({
+        id: keyRing.getNodeId().at(0),
```

Comment:
the id is so that MDNS can differentiate between packets sent by others and by itself, this is meant to be 16 bit, so i realized i should probably make it a sum of the id buffer

Reply:
16 bit prefix of node ID? I'm not sure but I think it's better to use a hashing function. However you won't get more than 256 nodes per local context. Can this be expanded to 32bits?